controllers/metallb_controller.go

```diff
@@ -177,7 +177,9 @@ func (r *MetalLBReconciler) syncMetalLBResources(config *metallbv1beta1.MetalLB)
 
 	for _, obj := range objs {
 		if err := controllerutil.SetControllerReference(config, obj, r.Scheme); err != nil {
```

Comment:
instead waiting for the error why not avoid setcontroller reference if namespace is "" continue ?

Reply:
:+1:

Reply:
right will change.